### PR TITLE
No issue: FE adjustments

### PR DESCRIPTION
**Pull request by Brankub**
**You can preview the changes within** `/login-as`, `/create-user`:

- [x] responsive behaviour tested
- [x] browser and device tested
- [x] doesn’t break anything else

**Additional information**:

image
image

Validation:
image
image

### CODE REVIEW

**Comment by lory87** [comment]
:+1:

**Comment by GaretJax** [comment]
BE LGTM

**Comment by Chive** [comment]
nooo

File: aldryn_sso/templates/aldryn_sso/login_screen_local.html [hunk 1]
```diff
@@ -13,8 +13,18 @@ <h3 class="page-status">Login</h3>
             {{ form.non_field_errors }}
             {% with field=form.user %}
             <div role="form" class="form-group">
-                <label for="{{ field.id_for_label }}" class="control-label sr-only">{{ field.label }}</label>
-                {{ field }}
+                <a href="/create-user" class="pull-right link-marketing">Add user</a>
```
Comment (Chive) [inline aldryn_sso/templates/aldryn_sso/login_screen_local.html]: no static urls please, use ``{% url 'aldryn_local_create_user' %}`` instead

**Pull request merged**